src/openMVG/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+include(GNUInstallDirs)
```

Comment:
I think we should do this include only once (only in the `primary` cmake the one in src/CMakeLists.txt)